packaging/linux/rpm/package_binaries.sh

```diff
@@ -114,12 +114,12 @@ export debian_arch=i386
 # found.  "psmisc" provides "killall", which is used in run_keybase.
 # "initscripts" provides "service", which is used to start atd in the
 # post-install.
-dependencies="Requires: at, fuse, libXss.so.1, /sbin/service, psmisc"
+dependencies="Requires: at, fuse, libXss.so.1, /sbin/service, psmisc, lsof"
```

Comment:
redhat distros come with `machinectl` for sure?

Reply:
Will check.